src/sentry/api/endpoints/event_details.py

```diff
@@ -23,9 +23,8 @@ def get(self, request, event_id):
         is the event as it appears in the Sentry database and not the event
         ID that is reported by the client upon submission.
         """
-        try:
-            event = Event.objects.get(id=event_id)
-        except Event.DoesNotExist:
+        event = Event.objects.from_event_id(event_id, None)
```

Comment:
nit: Same here, can you `project_id=None` that just to be extra clear to readers?